[2025-02-11T16:25:34.991+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T16:25:35.007+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T16:25:35.013+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T16:25:35.013+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-11T16:25:35.021+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_kaggle_data> on 2025-02-10 00:00:00+00:00
[2025-02-11T16:25:35.027+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=127) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T16:25:35.028+0000] {standard_task_runner.py:72} INFO - Started process 130 to run task
[2025-02-11T16:25:35.029+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_to_s3_pipeline', 'download_kaggle_data', 'scheduled__2025-02-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpbev3gunp']
[2025-02-11T16:25:35.030+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask download_kaggle_data
[2025-02-11T16:25:35.067+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data scheduled__2025-02-10T00:00:00+00:00 [running]> on host a4e66f9d6d3a
[2025-02-11T16:25:35.126+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_to_s3_pipeline' AIRFLOW_CTX_TASK_ID='download_kaggle_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T00:00:00+00:00'
[2025-02-11T16:25:35.127+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-11T16:25:35.145+0000] {logging_mixin.py:190} INFO - downloaded
[2025-02-11T16:25:35.145+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T16:25:35.152+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-11T16:25:35.153+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=kaggle_to_s3_pipeline, task_id=download_kaggle_data, run_id=scheduled__2025-02-10T00:00:00+00:00, execution_date=20250210T000000, start_date=20250211T162535, end_date=20250211T162535
[2025-02-11T16:25:35.213+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T16:25:35.236+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T16:25:35.237+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-11T17:48:45.942+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T17:48:45.973+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T17:48:45.982+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T17:48:45.985+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-11T17:48:45.997+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_kaggle_data> on 2025-02-10 00:00:00+00:00
[2025-02-11T17:48:46.020+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T17:48:46.026+0000] {standard_task_runner.py:72} INFO - Started process 114 to run task
[2025-02-11T17:48:46.025+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_to_s3_pipeline', 'download_kaggle_data', 'scheduled__2025-02-10T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmprl8_k8bq']
[2025-02-11T17:48:46.027+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask download_kaggle_data
[2025-02-11T17:48:46.078+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_to_s3_pipeline.download_kaggle_data scheduled__2025-02-10T00:00:00+00:00 [running]> on host c9167234c945
[2025-02-11T17:48:46.194+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_to_s3_pipeline' AIRFLOW_CTX_TASK_ID='download_kaggle_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T00:00:00+00:00'
[2025-02-11T17:48:46.198+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-11T17:48:46.287+0000] {s3_data.py:20} INFO - Downloading dataset wyattowalsh/basketball to /opt/***/data
[2025-02-11T17:48:46.357+0000] {s3_data.py:23} INFO - Dataset downloaded successfully.
[2025-02-11T17:48:46.371+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T17:48:46.463+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-11T17:48:46.466+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=kaggle_to_s3_pipeline, task_id=download_kaggle_data, run_id=scheduled__2025-02-10T00:00:00+00:00, execution_date=20250210T000000, start_date=20250211T174845, end_date=20250211T174846
[2025-02-11T17:48:46.734+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T17:48:46.861+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
